clickless_mouse.talon

```diff
@@ -14,4 +31,10 @@ settings():
     # when auto hide is active, option to prevent redisplay for minor motions
     user.clickless_mouse_prevent_redisplay_for_minor_motions = 0
 
-^click less mouse$: user.clickless_mouse_toggle()
\ No newline at end of file
+^click less mouse$: user.clickless_mouse_toggle()
+^(click less | clicker) enable$: user.clickless_mouse_enable()
+^(click less | clicker) disable$: user.clickless_mouse_disable()
+
+# command specific to single_stage
+^(click less | clicker) {user.clickless_mouse_action}$: 

```

Comment:
I wonder if we should have tags for when the single vs two stage clicker is active, and split this command out into a separate talon file? This way, it's only active when appropriate.